src/components/Property/Property.js

```diff
@@ -43,12 +43,13 @@ export default class Property extends PureComponent {
 
     return (
       <tr
-        className={classNames(classes.property, {
-          [classes.last]: isLast
+        className={classNames(classes.property, 'property', {
```

Comment:
We cant have generic classes if we want to commit to JSS. 

I suspect the solution is to restructure the DOM we generate, and to treat `@global` stuff completely separately to our own classes - or to attach our own classes to their elements where possible.